src/padplus-manager/padplus-manager.ts

```diff
@@ -591,8 +591,10 @@ export class PadplusManager extends EventEmitter {
             const deleteUserName = contactData.field
             if (this.cacheManager) {
               if (isRoomId(deleteUserName)) {
-                await this.cacheManager.deleteRoomMember(deleteUserName)
-                await this.cacheManager.deleteRoom(deleteUserName)
+                setTimeout(async () => {
+                  await this.cacheManager!.deleteRoomMember(deleteUserName)
```

Comment:
Would a flag be better here to mark the room as deleted?

Reply:
Done!